apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Tests/GoogleStackdriverTargetTest.cs

```diff
@@ -163,6 +163,26 @@ private LogEventInfo CreateLog(LogLevel level, string msg)
             Assert.Equal("Milky way", entry0.Labels["Galaxy"]);
         }
 
+        [Fact]
+        public async Task SingleLogEntryWithJsonProperties()
+        {
+            var uploadedEntries = await RunTestWorkingServer(
+                googleTarget =>
+                {
+                    googleTarget.SendJsonPayload = true;
+                    googleTarget.ContextProperties.Add(new TargetPropertyWithContext() { Name = "Galaxy", Layout = "Milky way" });
+                    LogManager.GetLogger("testlogger").Info("Hello {Planet}", "Earth");
+                    return Task.FromResult(0);
+                }, includeEventProperties: true);
+            Assert.Single(uploadedEntries);
+            var entry0 = uploadedEntries[0];
+            Assert.Equal(string.Empty, entry0.TextPayload?.Trim() ?? string.Empty);

```

Comment:
Please use `""` rather than `string.Empty`. We don't use `string.Empty` anywhere else in this repo.

Reply:
Fixed